src/Foundation/Features/Checkout/Checkout.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model CheckoutViewModel
+
+@using (Html.BeginForm("PlaceOrder", "Checkout", FormMethod.Post, new { @class = "row jsCheckoutForm", id = "jsCheckoutForm", novalidate = "novalidate" }))
+{
+    @Html.AntiForgeryToken()
+    <div class="col-md-8 col-xs-12">
+        <div class="card items">
+            <div class="card-body">
+                @Html.Partial("ShippingInformation", Model)
+                <br />
+                @Html.Partial("SingleAddress", Model)
```

Comment:
I think the ShippingInformation and Single Address views need to be combined to properly support multiple shipments.  The new view should loop through each shipment like before.  Please update the view with the following

- Add a split shipment button after Quantity text box that is only enabled when quantity is greater than one
- When the split button is clicked a new shipment will be created with 1 quantity removed the original shipment and added to the new shipment.
- Each shipment should have an address selector for that particular shipment.  Please add border around each shipment and separate shipping section with a border as well to show to user the section is for shipping information
- Remove the use billing address as shipping address
- Add a button for for entering new address to update the order when the new address is entered
- When changing address in the drop down make sure to update the order with the new address
- With these changes shipping address information should be updated before going to the next billing section